app/models/container_build.rb

```diff
@@ -0,0 +1,14 @@
+class ContainerBuild < ActiveRecord::Base
+  include CustomAttributeMixin
+  include ReportableMixin
```

Comment:
@moolitayer I tried but apparently we need all three of those for some of the screens in the UI.